### DISCUSSION TITLE
How to enable ccs phonetic conversion in LibreSVIP?

### DISCUSSION THREAD

**Discussion by Eataaa** [SoulMelody/LibreSVIP#617] [Q&A]
Hi, 
Is that possible to enable phoneme support for Cevio files? If it's possible, how do i enable it?
Thank you!

**Comment by SoulMelody** [comment]
CeVIO's phoneme format is different from the others. ccs's phonemes are in comma-separated format. Here's a [mapping table of Japanese romaji to CeVIO phonemes](https://github.com/SoulMelody/LibreSVIP/blob/main/libresvip/core/lyric_phoneme/japanese/cevio_romaji_mapping.py), is that what you're referring to?

**Reply by Eataaa** [reply]
Thank you for your reply,
Nope, that was not the one i was referring to. 
i want to convert **exact** phonemes from a file like svp to ccs.
For example, if i convert a note in synthv that has `"phonemes": "o n i gy i ry i"` to Cevio, the ccs file will have `Phonetic="o n i gy i ry i"` as well
Btw, the phoneme in `Phonetic=" "` field in ccs file doesn't have to be separated by comma, it can be separated by space as well(like other vocal synths)
